### DISCUSSION TITLE
v1.15.1

### DISCUSSION THREAD

**Discussion by Kovah** [Kovah/LinkAce#829] [Announcements]
## What's Changed
* Reduce release image size by @rare-magma in https://github.com/Kovah/LinkAce/pull/799
* Dependencies were updated.

## New Contributors
* @rare-magma made their first contribution in https://github.com/Kovah/LinkAce/pull/799

**Full Changelog**: https://github.com/Kovah/LinkAce/compare/v1.15.0...v1.15.1

<hr /><em>This discussion was created from the release <a href='https://github.com/Kovah/LinkAce/releases/tag/v1.15.1'>v1.15.1</a>.</em>

**Comment by WorthStudy** [comment]
May I ask how non Docker applications can upgrade after downloading and decompressing the latest version of the distribution package？

**Reply by Kovah** [reply]
There's a guide in the docs: https://www.linkace.org/docs/v1/upgrade/#upgrade-a-non-docker-installation

Please let me know if there are issues with it.

**Reply by WorthStudy** [reply]
Okay, thank you, it's already resolved!
